domainservice/src/main/kotlin/com/makarytskyi/rentcar/kafka/CreateRepairingKafkaProcessor.kt

```diff
@@ -0,0 +1,30 @@
+package com.makarytskyi.rentcar.kafka
+
+import com.makarytskyi.internalapi.commonmodels.repairing.Repairing
+import com.makarytskyi.rentcar.mapper.OrderMapper.toNotification
+import com.makarytskyi.rentcar.service.OrderService
+import com.makarytskyi.rentcar.util.timestampToDate
+import jakarta.annotation.PostConstruct
+import org.springframework.stereotype.Component
+import reactor.kafka.receiver.KafkaReceiver
+
+@Component
+class CreateRepairingKafkaProcessor(
+    val kafkaReceiver: KafkaReceiver<String, ByteArray>,
+    val userNotificationKafkaProducer: UserNotificationKafkaProducer,
+    val orderService: OrderService
+) {
+
+    @PostConstruct
```

Comment:
What if the application context fails to start after creating your processor bean? 
Is it correct that the not-ready application will consume traffic?